src/MongoDB.Analyzer/Core/Poco/PocoDataFiller.cs

```diff
@@ -116,7 +116,7 @@ private static object HandlePrimitive(string memberName, Type memberType)
     private static object HandleSystemType(Type systemType, string memberName) =>
         systemType.FullName switch
         {
-            "System.DateTime" => new DateTime(memberName.Length * 100, memberName.Length % 12, memberName.Length % 30),
+            "System.DateTime" => new DateTime(1900 + memberName.Length % 150, memberName.Length % 12, memberName.Length % 28, 0, 0, 0, 0, DateTimeKind.Utc),
```

Comment:
DateTime constructor parameters month and day should be greater or equal to 1. But ("12-char-name".Length % 12) would result to 0 - which is out of range. Fix the corner case or consider usage of simpler code to generate random date:

```
private static DateTime GenerateDate(string memberName)
	=> new DateTime(2000, 1, 1).AddSeconds(memberName.GetHashCode()).Date;
```

Reply:
Discussed offline.